src/Uno.UI.Runtime.Skia.MacOS/UnoNativeMac/UnoNativeMac/UNOWindow.m

```diff
@@ -716,6 +716,7 @@ - (void)sendEvent:(NSEvent *)event {
                 }
             }
 #endif
+            data.pointerDeviceType = pdt;
```

Comment:
@jeromelaban, @spouliot wondering if we can have some tests?

Reply:
Not at this layer. A manual test would be preferrable, if none already exist.